pallets/parachain-staking/src/lib.rs

```diff
@@ -611,6 +611,9 @@ pub mod pallet {
 				candidates: vec![],
 				delegations: vec![],
 				inflation_config: Default::default(),
+				collator_commission: Default::default(),
+				parachain_bond_reserve_percent: Default::default(),
+				blocks_per_round: Default::default(),
```

Comment:
I think it would be good to require these to be explicitly specified instead of these unusable default values.

Right now, I suspect that these defaults here would cause problems downstream (e.g. `blocks_per_round == 0` will cause a panic somewhere). It would be preferable to see a message like "error: no value provided for genesis".

Along with that error could be a comment suggesting that the values used in genesis should be the ones previously specified in the runtime config.

Reply:
`Default` impl of GenesisBuild isn't used in our code so it shouldn't matter but I'll change `blocks_per_round: 1u32` and add an assert in genesis build to check that it is greater than 0.